source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -10,4 +10,47 @@ Rename Diagrams
    :depth: 1
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the Mapping tab of Relational Migrator.
+Relational Migrator displays diagram names on the ribbon tab. 
```

Comment:
[question] Small question since I'm not as familiar with Relational Migrator, but is the ribbon tab an apparent visual element and/or is it called the "Ribbon" tab similar to the "Mapping" tab? Do you think this is something that users might benefit from with a screenshot or is it better as is?

Just asking this since I see the ribbon tab being  referenced a handful of times on the page.

Reply:
It is a visual element (not an official tab name like "Mapping") but I'm going to cc @ianf-mongodb here to check if this term should be used here.

Reply:
Hi @ajhuh-mdb looking into this, we typically refer to navigation tabs at the top (Mapping/ Code Generation / Data Migration) as a "screen". Aka: `From the :guilabel:`Mapping` screen`...., I did look to our [UI overview page ](https://www.mongodb.com/docs/relational-migrator/projects/overview/) we refer to the diagram controls as a "Diagram Toolbar". On the create diagram task page we do have a screenshot of the UI for the diagrams control. I think we could:

1. Include an additional screenshot for step 1, showing the toolbar to center the user.
2. Refer to the UI element as `Diagram Toolbar` to match the UI Overview screen.  

![Screenshot 2024-01-11 at 1 50 20 PM](https://github.com/mongodb/docs-relational-migrator/assets/85948430/ca93d5cf-fc48-4eb2-85e6-08a00464384b)

Reply:
That makes sense, thanks @ajhuh-mdb and @ianf-mongodb ! Just implemented those changes